src/fen_to_graph/pieces.clj

```diff
@@ -41,10 +41,8 @@
 
 (defn- pawn-initial? [color position]
   "Check if a pawn is in its initial position."
-  (cond
-    (and (= color :white) (= (second (name position)) \2)) true
-    (and (= color :black) (= (second (name position)) \7)) true
-    :else false))
+  (let [initial-position {:white \2 :black \7}]
+    (= (second (name position)) (initial-position color))))
```

Comment:
I know I could inline initial-position, but it's nice to give a name to things ;)